recipe/conda_build_config.yaml

```
@@ -0,0 +1,15 @@
+c_compiler_version:
+  - 10
+cxx_compiler_version:
+  - 10
+cuda_compiler_version:
+  - 11.6
+fortran_compiler_version:
+  - 10
+cdt_name:
```

Would add this here as well as it is needed to meet the requirements of this [`zip_keys`]( https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/f16156d7aea59cb6cc51a5d6708ab019d973b670/recipe/conda_build_config.yaml#L153-L160 ) section

```suggestion
cudnn:
  - 8
cdt_name:
```

---

> Would add this here as well as it is needed to meet the requirements of this [`zip_keys`](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/f16156d7aea59cb6cc51a5d6708ab019d973b670/recipe/conda_build_config.yaml#L153-L160) section

Will this mark dependency on cudnn? The ptxcompiler build doesn't depend on cudnn.

---

Good question, but no it will be ignored. It's only needed for the `zip_keys` line to work correctly